docs/dev_guide/migrate-a-project/generate-compilation-db.rst

```diff
@@ -115,6 +115,20 @@ in the build directory:
 
    CMake 3.10 or higher is recommended as it provides better support for CUDA source code.
 
+Generate a Compilation Database for Setuptools-Based Projects
+--------------------------------------------------------
+Setuptools is a package development library in Python for facilitating packaging, distribution and installation of Python projects.For projects with setuptools as their build system, the steps to generate a compilation database are as follows:
+
+#. Generate the build log for the project using the verbose flag 
+#. Use ``intercept-build –-parse-build-log`` option to generate the compilation database.

```

Comment:
Line 122 should have a period at the end to match line 123.

Reply:
Done. Thanks.